src/index.ts

```diff
@@ -17,11 +17,11 @@ const confippet = {
   providerTypes: providerTypes,
   extHandlers: extHandlers,
   util: util,
-  loadConfig: loadConfig
+  loadConfig: loadConfig,
 };
 
 Object.defineProperty(confippet, "config", {
-  get: () => require("../config")
+  get: () => require("../config"),
```

Comment:
why is this change required? it is breaking my build.since there is no config inside the src directory

Reply:
Error: Cannot find module '../config'
Require stack:
- node_modules/electrode-confippet/lib/index.js

getting this error

Reply:
publish config issue.  fixed in 1.7.1, thanks.